base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -450,4 +461,19 @@ func (vgm *virtualGroupManager) releaseSPAndGVGLoop() {
 	for range ticker.C {
 		vgm.freezeSPPool.ReleaseSP()
 	}
+	vgm.mutex.RLock()
+	aliveSP := make([]*sptypes.StorageProvider, 0)
+	for _, sp := range vgm.spManager.otherSPs {
+		if sp.IsInService() {
+			aliveSP = append(aliveSP, sp)
+		}
+	}
+	vgm.mutex.RUnlock()
+	params, err := vgm.chainClient.QueryStorageParamsByTimestamp(context.Background(), time.Now().Unix())
+	if err != nil {
+		return
```

Comment:
I think we should never return, otherwise no SP will be unfreeze from the pool any more.